python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -151,3 +151,23 @@ def __init__(self, max_brightness=0, max_contrast=0, max_saturation=0, max_hue=0
 
     def hybrid_forward(self, F, x):
         return F.image.random_color_jitter(x, *self._args)
+
+
+class AdjustLighting(HybridBlock):
+    def __init__(self, alpha_rgb=(0.0, 0.0, 0.0), eigval=(55.46, 4.794, 1.148),
```

Comment:
what's alpha_rgb?
use base._Null for default arguments to avoid parsing

Reply:
It's (alpha_r, alpha_g, alpha_b)